<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="GGKU5DELL1725" failures="2" tests="2" name="Test(failed)" time="0.04" errors="0" timestamp="2022-09-21T17:17:02 IST">
  <testcase classname="RetryLogicDemo.TestcasesExample" name="TestCase02" time="0.006">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at RetryLogicDemo.TestcasesExample.TestCase02(TestcasesExample.java:16)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- TestCase02 -->
  <testcase classname="RetryLogicDemo.TestcasesExample" name="TestCase01" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at RetryLogicDemo.TestcasesExample.TestCase01(TestcasesExample.java:10)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- TestCase01 -->
  <testcase classname="RetryLogicDemo.TestcasesExample" name="TestCase03" time="0.0">
    <ignored/>
  </testcase> <!-- TestCase03 -->
</testsuite> <!-- Test(failed) -->
